pkg/operator/operator_test.go

```diff
@@ -412,12 +412,12 @@ func TestRebalanceWorkloads(t *testing.T) {
 	)
 
 	for _, tc := range []struct {
-		name             string
-		pvs              []v1.PersistentVolume
-		pvcs             []v1.PersistentVolumeClaim
-		spreadByOperator bool
-		expectedPods     []string
-		expectedPVCs     []string
+		name               string
+		pvs                []v1.PersistentVolume
+		pvcs               []v1.PersistentVolumeClaim
+		balancedByOperator bool
+		expectedPods       []string
+		expectedPVCs       []string
```

Comment:
should we add some assertions about cordoning nodes?

Reply:
we can't really tell whether the node was cordoned or not here, the only thing we can assert is that the node on which the pods are is schedulable. This assertion is already present, at the end of the test function.